### ISSUE TITLE
Contrast issue with the background-color of <code> elements

### ISSUE THREAD

**Issue by glmvc** [mdn/yari#10954]
### Summary

In both light and dark mode, the background color of the `<code>` elements can hardly be distinguished from the background color behind it in certain places.

For example, take a look at the zebra-striped "Technical summary" tables, where each odd row has a `background-color` that is nearly identical to the one of the `<code>` elements within it.

### URL

The issue is global (styling).
Example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/colgroup#technical_summary

### Reproduction steps

The issue is visual (styling).
Please visit the sample URL or check out the screenshots below.

### Expected behavior

A distinguishable `background-color` to visually recognize the `<code>` elements.

### Actual behavior

The `<code>` elements are visually indistinguishable because of the nearly identical `background-color`.

### Device

Desktop

### Browser

Chrome

### Browser version

Stable

### Operating system

Mac OS

### `#f2f1f1` on `#f9f9fb` (Contrast Ratio = 1.07:1)
![mdn-technical-summary-light](https://github.com/mdn/yari/assets/95487764/21052b9a-0dc1-4c76-bd52-7c889ba09d59)

### `#343434` on `#313131` (Contrast Ratio = 1.04:1)
![mdn-technical-summary-dark](https://github.com/mdn/yari/assets/95487764/f076e8e6-b11d-4603-9b79-20868132a3b3)

### Anything else?

[CSS that sets the background-color for `<code>`](https://github.com/mdn/yari/blob/d8173cc190686634cd43f0a83b4f6178a133e9f5/client/src/document/index.scss#L460)
[CSS that sets the background-color for table cells in odd table rows](https://github.com/mdn/yari/blob/d8173cc190686634cd43f0a83b4f6178a133e9f5/client/src/document/index.scss#L341)

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by skyclouds2001** [comment]
this is a self-link (a kind of link in a page that reference to the page itself)

**Comment by glmvc** [comment]
@skyclouds2001 You're right about `<colgroup>`, but note the `<col>` link, for example. And it's possible that other links appear in such tables. The "Technical summary" table is just one example.

**Comment by caugner** [comment]
This looks better now:

| Light | Dark |
|--------|--------|
| <img width="780" height="304" alt="Image" src="https://github.com/user-attachments/assets/da04e750-ee19-4ed0-ad67-913427564dc0" /> | <img width="780" height="304" alt="Image" src="https://github.com/user-attachments/assets/8d74b1cf-b9bc-4edb-b328-67ad07af0aba" />|

The contrast ratio certainly isn't high, but it's possibly the best we can do, without compromising the contrast against the text.

Given that the code background color stands in combination with the font, it is dispensable to some degree, so I consider this issue resolved.
